Week1/homework/4-Log the string.js

```diff
@@ -0,0 +1,19 @@
+'use strict'
+
+const myString = 'wesam alkfere';
+
+console.log(' the value will be wesam alkfere'); 
+
+console.log(myString);
+
+const myString ='zaandam';
+
+console.log(' has already been declared.. can not a new value');
```

Comment:
Only now I see this comment. So you did try to run the code and got an error. You have to make some changes in order to make it work.